sources/packages/backend/apps/db-migrations/src/sql/BetaUsersAuthorizations/Create-beta-users-authorizations-table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS beta_users_authorizations(
+  id SERIAL PRIMARY KEY,
+  given_names VARCHAR(100) NULL,
+  last_name VARCHAR(100) NOT NULL,
+  enabled_from TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  CONSTRAINT beta_users_authorizations_name_unique UNIQUE (given_names, last_name)
+);
+
+-- ## Comments
+COMMENT ON TABLE beta_users_authorizations IS 'Beta users authorizations table. Only users with a BCSC credential that matches given name and last name in the period starting from `enabled_from` column will be allowed during the beta user period.';
```

Comment:
The back ticks will not affect the format. Please change it to single quotes or removed it (`'enabled_from'`).